tests/test_watson.py

```diff
@@ -794,6 +794,26 @@ def test_report(watson):
     with pytest.raises(WatsonError):
         watson.report(arrow.now(), arrow.now(), tags=["A"], ignore_tags=["A"])
 
+    watson.start('foobaz')
+
+    report = watson.report(
+        arrow.now(), arrow.now(), current=True, projects=['foobaz']
+    )
+    assert len(report['projects']) == 1
+    assert watson.frames['id'].count('current') == 0
+
+    report = watson.report(
+        arrow.now(), arrow.now(), current=False, projects=['foobaz']
+    )
+    assert len(report['projects']) == 0
+
+    report = watson.report(
+        arrow.now(), arrow.now(), projects=['foobaz']
+    )
+    assert len(report['projects']) == 0
+
+    watson.stop()
```

Comment:
Finally, something I missed in my initial review: this seems a great opportunity to split `test_report()` in multiple tests with more meaninful names. For example, your new tests could be inside a method named `test_report_current()`.